vendor/mobify-js/1.1.2

```diff
@@ -0,0 +1 @@
+Subproject commit b786b00873bf01cb4fe5f548a37938a85cfc7082
```

Comment:
You're right, fixed: https://github.com/mobify/mobify-client/commit/131c76e250f740d2a879df2686a5fddeddc0aa15 https://github.com/mobify/mobify-client/commit/b78dd298821947fb93f3c4efff9b94d7ec7fa72f